templates/_header.html

```diff
@@ -16,7 +16,14 @@
       <ul class="p-navigation__links" role="menu">
         <li class="p-navigation__link" role="menuitem">
           <a
-            {% if (path and (path.startswith('/store') or path.startswith('/search'))) or snap_title %} class="is-selected"{% endif %}
+            {% if
+              path and (
+                (
+                  path.startswith('/store') or path.startswith('/search')
+                ) or (
+                  snap_title and not path.startswith('/account')
+                )
+              ) %} class="is-selected"{% endif %}
```

Comment:
Works for me but it seems that there could be a much easier way of doing this

Reply:
Yeah, this has got a bit complicated now tbh, but at least it is automatic.
I didn't really like 'page_slug' as it depended on adding the right string to the views context for each page. It seemed more error prone. Using the built-in flask stuff is more reliable then humans IMO

Reply:
Now it's a complicated logic that depends on route path name (that can potentially change) and existence of snap_title in snap details (but in theory it can be used on any other page as well - as the example shows).

Reply:
Maybe we could at least move this logic out of template to context processor, so it's automatically added to all templates? So it would be one place that would have the logic for creating something like `page_slug` (instead of doing it manually)?

Reply:
Yeah that makes sense - generate the page_slug based on some rules that are in a specific file... It either does something special (as in this case) or just passes through the flask path.